nikola/__main__.py

```diff
@@ -241,14 +241,15 @@ def load_tasks(self, cmd, opt_values, pos_args):
         if self.quiet:
             DOIT_CONFIG = {
                 'verbosity': 0,
-                'reporter': 'zero',
+                'reporter': 'zero'
```

Comment:
No need to change it, but I tend to leave the last commas in this sort of things because it makes the diff better the next time something is added :-)